test/.eslintrc.yml

```diff
@@ -1,4 +1,8 @@
+parserOptions:
+  ecmaVersion: 5 # don't rely on default, since its changed by env: es6
```

Comment:
It should be fine if our tests use es6 features, we only don't want our sources to use es6 feature because we don't want to integrate babel in v2.